expertise/create_dataset.py

```diff
@@ -54,13 +54,32 @@ def get_paper_notes(self, author_id, dataset_params):
         notes_v2 = list(openreview.tools.iterget_notes(self.openreview_client_v2, content={'authorids': author_id}))
         return notes_v1 + notes_v2
 
+    def deduplicate_publications(self, publications):
+        deduplicated = []
+
+        # Build index of pub.original
+        original_ids = [getattr(pub, 'original', None) for pub in publications]
```

Comment:
Here is what I have now: https://github.com/openreview/openreview-expertise/blob/9501e0eda62958005434ff3daf5f0394aef1ed69/expertise/create_dataset.py#L61

Do you mean to make a change like the following:
`original_ids = [pub.original if pub.original is not None for pub in publications] `
to avoid repeatedly comparing against None values?

Reply:
Yes, but I also mean that you could use a [set](https://docs.python.org/3/tutorial/datastructures.html#sets). This will make it more efficient when you check for duplicates.